src/boot/parse-json-config.js

```diff
@@ -1,5 +1,10 @@
-// `Object.assign()`-like helper. Used because this script needs to work
-// in IE 10/11 without polyfills.
+/**
+ * `Object.assign()`-like helper. Used because this script needs to work
+ * in IE 10/11 without polyfills.
```

Comment:
At the time we dropped IE support, the boot script was designed to still run in IE but log a console error instead of crashing. We've dropped IE support for a couple of years now, so we could probably look at dropping support for it in the boot script.